services_delegate.go

```diff
@@ -165,17 +180,36 @@ func (d *servicesDelegate) NotifyUpdate(node *memberlist.Node) {
 	log.Debugf("NotifyUpdate(): %s", node.Name)
 }
 
+// Try to pack as many messages into the packet as we can. Note that this
+// assumes that no messages will be longer than the normal UDP packet size.
+// This means that max message length is somewhere around 1398 when taking
+// messaging overhead into account.
```

Comment:
Yes, but without them being named, it's not clear which is which since they are the same data type.